chia/consensus/blockchain_interface.py

```diff
@@ -24,15 +24,21 @@ def height_to_block_record(self, height: uint32) -> BlockRecord: ...
     async def prev_block_hash(self, header_hashes: List[bytes32]) -> List[bytes32]: ...
 
 
-class BlockchainInterface(BlockRecordsProtocol, Protocol):
+class FullBlocksProtocol(BlockRecordsProtocol, Protocol):
```

Comment:
i think since we deal with block records as well i would call this BlocksProtocol

Reply:
makes sense

Reply:
done